### DISCUSSION TITLE
How to recieve files from the chatbot?

### DISCUSSION THREAD

**Discussion by harsdua** [tjtanjin/react-chatbotify#182] [Q&A] [answered]
Hello, 
I have wired up the chatbot to talk with my API. I would like to recieve files but I cant find any documentation regarding this. Could you please help me out :c 🙏 ?

**Comment by tjtanjin** [comment, marked as answer]
Hey @harsdua, if your download is a link, you can enable markup language in `settings.botBubble.dangerouslySetInnerHtml`. Alternatively, you can use the `component` attribute to add your own component for download: https://react-chatbotify.com/docs/api/attributes#component

**Reply by harsdua** [reply]
Thanks for the quick reply! Great work and idea btw.

**Reply by tjtanjin** [reply]
> Thanks for the quick reply! Great work and idea btw. 

No problem! Feel free to reach out if you need more clarifications 😊
